### PR TITLE
Introduce yourself

### PR DESCRIPTION
**Pull request by AsmaKacem1**
Introduce yourself : Asma-intro

### CODE REVIEW

File: posts/Asmaa-intro/index.qmd [hunk 1]
```diff
@@ -1,11 +1,41 @@
 ---
-title: ""
-author: "Asmaa Kasim"
+title: "Outreachy - Introduce yourself"
+author: "Asma Kacem"
 date: "2022-05-30"
 categories: [outreachy]
-draft: true
+
```
Comment (BatoolMM) [inline posts/Asmaa-intro/index.qmd:6]: subtitle: "A little bit about myself"
license: "CC BY"
toc-title: Table of contents
toc-location: left

File: posts/Asmaa-intro/index.qmd [hunk 2]
```diff
@@ -1,11 +1,41 @@
 ---
-title: ""
-author: "Asmaa Kasim"
+title: "Outreachy - Introduce yourself"
+author: "Asma Kacem"
```
Comment (BatoolMM) [inline posts/Asmaa-intro/index.qmd:3]:
```suggestion
author:
  - name: Asmaa Kasim 
    url: https://asmakacem.me/
    affiliation: National Engineering School of Sousse (ENISO)
 ```

**Review by BatoolMM — APPROVED** [review]

File: posts/Asmaa-intro/index.qmd [hunk 3]
```diff
@@ -1,11 +1,44 @@
...
-author: "Asmaa Kasim"
+title: "Outreachy - Introduce yourself"
+author:
+  - name: Asmaa Kasim 
+    url: https://asmakacem.me/
+    affiliation: National Engineering School of Sousse (ENISO)
 date: "2022-05-30"
 categories: [outreachy]
-draft: true
+
```
Comment (BatoolMM) [inline posts/Asmaa-intro/index.qmd:9]:
```suggestion
subtitle: "A little bit about myself"
license: "CC BY"
toc-title: Table of contents
toc-location: left
```

**Review by BatoolMM — APPROVED** [review]

**Comment by BatoolMM** [comment]
Thank you, Asmaa - The blog is live now on the website, I made a few modifications for the images, I hope you don’t mind!

**Comment by AsmaKacem1** [comment]
> Thank you, Asmaa - The blog is live now on the website, I made a few modifications for the images, I hope you don’t mind!

Thank you very much Batool I like the modification of images, I think it's better now. I just make a modification of my name.

**Comment by BatoolMM** [comment]
Just fixed the name, sorry for that!

**Pull request merged**